Vagrantfile

```diff
@@ -27,11 +27,8 @@ Vagrant.configure("2") do |config|
   config.vm.provision provisioner_type do |ansible|
     ansible.playbook = "ansible/vagrant-playbook.yml"
     ansible.galaxy_role_file = "ansible/requirements.yml"
-    # Ansible has a bug where the "--module-path" option is not respected
-    # ansible.raw_arguments = ["--module-path=" + File.expand_path("ansible/library")]
-    ENV["ANSIBLE_LIBRARY"] = File.expand_path("ansible/library")
```

Comment:
Let's leave these lines about setting the `ENV`, as it's still necessary for ansible running on the host. The config file points to a path that only exists within the guest.